### PR TITLE
feat(helm): cleanup chart's unused components

### PR DESCRIPTION
**Pull request by IGZangelcostales**
## Description
Remove the following components from the chart:

* Chronograf templates and values
* InfluxDB from chart’s dependencies and values
* Kapacitor from chart’s dependencies and values
* Mongo-Express from templates and values
* Credentials secrets for Mongo-Express in the local dev mongodb creation (helmfile)

Finally, move all the templates folders from the `engine` folder to the chart's root folder.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [x] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by alpiquero — APPROVED** [review]

File: helm/kai/templates/mongo-writer/deployment.yaml
Comment (dfinteligenz) [inline helm/kai/templates/mongo-writer/deployment.yaml:1]: We should be able to remove the mongo-writer as we are not using it anymore.

File: helm/kai/README.md [hunk 1]
```diff
@@ -26,9 +26,13 @@ $ helm install [RELEASE_NAME] konstellation-io/kai
...
 
-By default this chart installs [InfluxDB](https://github.com/influxdata/helm-charts/tree/master/charts/influxdb) and [Kapacitor](https://github.com/influxdata/helm-charts/tree/master/charts/kapacitor) chart as dependency.
+* minio
+* grafana
+* loki
+* prometheus
+* redis
 
-However, **Kapacitor** is an optional dependency. To disable a it during installation, set `kapacitor.enabled`, to `false`.
+except for **minio** everything else is an optional dependency. To disable it during installation, set `<component_name>.enabled`, to `false`.
```
Comment (dfinteligenz) [inline helm/kai/README.md:35]: I'd add a bit more of context here.

The components are not optional, but externalizable. If disabled, the user needs to set the needed config for using these components from their infrastructure.

**Review by dfinteligenz — CHANGES_REQUESTED** [review]

File: helm/kai/templates/mongo-writer/deployment.yaml
Comment (dfinteligenz) [reply helm/kai/templates/mongo-writer/deployment.yaml:1]: https://github.com/konstellation-io/kai/pull/202

File: helm/kai/README.md (hunk 1 above)
Comment (IGZangelcostales) [reply helm/kai/README.md]: Done

**Pull request merged**